_internal/build.yaml

```diff
@@ -1,17 +0,0 @@
-targets:
```

Comment:
We should move this file into `/packages/core`

Reply:
I understood completely.